cuda_core/tests/test_graph.py

```diff
@@ -41,7 +43,9 @@ def _common_kernels_conditional():
                                                                                     unsigned int value);
     __global__ void empty_kernel() {}
     __global__ void add_one(int *a) { *a += 1; }
-    __global__ void set_handle(cudaGraphConditionalHandle handle, int value) { cudaGraphSetConditional(handle, value); }
+    __global__ void set_handle(cudaGraphConditionalHandle handle, bool value) {
```

Comment:
I think this was already wrong 🙃 `cudaGraphSetConditional` takes an `unsigned int`
```suggestion
    __global__ void set_handle(cudaGraphConditionalHandle handle, unsigned int value) {
```

Reply:
The new test does seem to be more robust! Thanks, Mike.